akka-docs/src/test/java/jdocs/stream/IntegrationDocTest.java

```diff
@@ -693,5 +695,28 @@ public void illustrateOrderingAndParallelismOfMapAsyncUnordered() throws Excepti
     };
   }
 
+  @Test
+  public void illustrateSourceQueue() throws Exception {
+    new TestKit(system) {
+      {
+        //#source-queue
+        int bufferSize = 5;
+        int elementsToProcess = 3;
+
+        SourceQueueWithComplete<Integer> sourceQueue = Source.<Integer>queue(bufferSize, OverflowStrategy.backpressure())
```

Comment:
Looks like a very wide row, does it really fit in the sample box in the docs? Break into multiple lines?

Reply:
ok